src/taskProvider.ts

```diff
@@ -338,8 +351,13 @@ export class TaskProvider implements vscode.TaskProvider {
   private appDeleteExec(): string {
     let exec = "";
     if (platform === "linux") {
+      let scpconfig = "";
+      const conf = vscode.workspace.getConfiguration("ledgerDevTools");
```

Comment:
We get the conf in the constructor and the `generateTasks` method, you don't need to have a conf variable here. Cf. 
https://github.com/LedgerHQ/ledger-vscode-extension/blob/23d15494c22d29ab3e26d4cc05b63c754fab08f1/src/taskProvider.ts#L177

So you can have a shared variables between task builders.

Reply:
The generateTasks method is called each time the settings are changed by the user, so we are sure the tasks use the latest setting.

Reply:
Fixed!